doc/dirvish.txt

```diff
@@ -165,6 +165,20 @@ can be overridden by handling that event. Example: >
         \ gh :silent keeppatterns g@\v/\.[^\/]+/?$@d _<cr>:setl cole=3<cr>
     augroup END
 <
+g:dirvish_shdo_before = ''                          *g:dirvish_shdo_before*
+    Inserts this text at the beginning of shell scripts generated with
+    |:Shdo|. The current dirvish directory is inserted wherever {} appears in
+    |g:dirvish_shdo_before|.
+    To ensure scripts always cd to the current dirvish directory, you could
```

Comment:
what is motivating this example?  :Shdo buffer already automatically sets (:lcd) its local-directory to the Dirvish directory.

Reply:
Three reasons:
* Using something like 'autochdir' (#19) or `au BufReadPost * silent! cd %:p:h` changes the shdo script cwd. In the latter case, I could block it but would have to block every temp script (`*.tmp.bat` or `$TEMP/*`)?
*  I'm not comfortable running scripts that rely on cwd. Instead of manually adding the path when doing something destructive like `:%Shdo rm {}`, this option automatically makes it explicit.
* Gives a nice entry point back to the directory. If I `:only`, edit the script, finish with it, I can `gf` on the path since using `-` brings me to $TEMP (as expected).